src/screens/ChangePasswordScreen/index.tsx

```diff
@@ -0,0 +1,114 @@
+import { LockOutlined as LockOutlinedIcon } from '@mui/icons-material';
+import { LoadingButton } from '@mui/lab';
+import { Avatar, Box, Container, TextField, Typography } from '@mui/material';
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { useSearchParams } from 'react-router-dom';
+
+import useChangePassword from '@src/operations/mutations/resetPassword';
+import usePasswordRules from '@src/screens/ChangePasswordScreen/hooks/usePasswordRules';
+
+export type ChangePasswordFormInput = {
+  password: string;
+  passwordConfirmation: string;
+};
+
+const ChangePasswordScreen: React.FC = () => {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+    getValues,
+  } = useForm<ChangePasswordFormInput>();
+  const [errorMessages, setErrorMessages] = useState<string[]>([]);
+  const changePassword = useChangePassword(setErrorMessages);
+
+  const validatePassword = (): boolean => {
+    return getValues('password') === getValues('passwordConfirmation');
+  };
+
+  const { passwordRules, passwordConfirmationRules } =
+    usePasswordRules(validatePassword);
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const onSubmit = async ({ password }: ChangePasswordFormInput) => {
+    await changePassword({
+      variables: {
+        token: searchParams.get('token') || '',
+        password,
+      },
+    });
+  };
+
+  return (
+    <Container data-testid="loginScreen" component="main" maxWidth="xs">
```

Comment:
testid가 잘못 선언되어 있어보여요

```suggestion
    <Container data-testid="changePasswordScreen" component="main" maxWidth="xs">
```